spell/spell-reviewer-mainnet-checklist.md

```diff
@@ -70,6 +67,7 @@ Spell Actions:
 * [ ] Onboarding
   * [ ] [Collateral Onboarding](./collateral-onboarding-checklist.md)
   * [ ] [RWA Onboarding](./rwa-onboarding-checklist.md)
+  * [ ] [Teleport Onboarding](./teleport-oboarding-checklist.md)
```

Comment:
Typo again

Reply:
hmm, interesting, good catch! It seems that the CI didn't catch it

Reply:
done in c0b66d1